sparkle/CLI/generate_report.py

```diff
@@ -227,6 +259,21 @@ def parser_function() -> argparse.ArgumentParser:
             ablation_scenario = AblationScenario(
                 solver, instance_set_train, instance_set_test,
                 gv.settings().DEFAULT_ablation_output)
+
+        # Create machine readable output
+        # Where can this path variable be found?
+        path = \
+            Path("Output/Configuration/Raw_Data/SMAC2/scenarios/PbO-CCSAT-Generic_PTN")
```

Comment:
ReportingScenario object, you may have to reconstruct from "{solver_name}_{InstanceSetTrainName}" or something

Reply:
Yes I messaged you about that
I can reconstruct it but only if I do the remaining parts as a string (Raw_Data, scenario, etc.)
Is that okay?